tests/integration-tests/tests/create/test_create/test_create_wrong_os/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = slurm
+master_instance_type = {{ instance }}
+custom_ami = {{ custom_ami }}
+master_root_volume_size = 100
```

Comment:
do we need to explicitly set master and compute root volume size in these clusters?

Reply:
I am using deeping learning AMIs for runtime baking AMI. They require root volume at least 100GB